public/src/components/campaign/campaign_list.jsx

```diff
@@ -42,7 +36,7 @@ export default class CampaignList extends Component {
               button_collection={account_info.is_admin ?
                                    admin_campaigns :
                                    volunteer_campaigns}
-              componentClickHandler={handleClick}
+              componentClickHandler={updateCampaignStatus}
```

Comment:
Maybe check this against your current ticket. I think I remember this was changed for the campaign stats as well... just want to make sure the changes are compatible.

Reply:
@nguyenaiden see comment above. as long as we didn't delete the action call that assigns users to campaigns this is fine